assets/dev/js/frontend/utils/lightbox/lightbox-manager.js

```diff
@@ -0,0 +1,94 @@
+export default class extends elementorModules.ViewModule {
+	static getLightbox() {
+		const lightboxPromise = new Promise( ( resolveLightbox ) => {
+				import(
+					/* webpackChunkName: 'lightbox' */
+					`elementor-frontend/utils/lightbox/lightbox`
+				).then( ( { default: LightboxModule } ) => resolveLightbox( new LightboxModule() ) );
+			} ),
+			dialogPromise = elementorFrontend.utils.assetsLoader.load( 'script', 'dialog' ),
+			shareLinkPromise = elementorFrontend.utils.assetsLoader.load( 'script', 'share-link' );
+
+		return Promise.all( [ lightboxPromise, dialogPromise, shareLinkPromise ] ).then( () => lightboxPromise );
+	}
+
+	getDefaultSettings() {
+		return {
+			selectors: {
+				links: 'a, [data-elementor-lightbox]',
+			},
+		};
+	}
+
+	getDefaultElements() {
+		return {
+			$links: jQuery( this.getSettings( 'selectors.links' ) ),
+		};
+	}
+
+	isLightboxLink( element ) {
+		if ( 'A' === element.tagName && ( element.hasAttribute( 'download' ) || ! /^[^?]+\.(png|jpe?g|gif|svg|webp)(\?.*)?$/i.test( element.href ) ) ) {
```

Comment:
Fix according to:
https://github.com/elementor/elementor/pull/14063

Reply:
Fixed.